README.md

```diff
@@ -194,6 +194,32 @@ cargo clippy -- -A clippy::all -W clippy::useless_format -W clippy::...
 ```
 Note that if you've run clippy before, this may only take effect after you've modified a file or ran `cargo clean`.
 
+### Specifying the minimum supported Rust version
+
+Projects that intend to support old versions of Rust can disable lints pertaining to newer features by
+specifying the minimum supported Rust version (MSRV) in the clippy configuration file.
+
+```toml
+msrv = "1.30.0"
```

Comment:
IIUC the #6201 implementation does not consider the exact patch version, so I think it is preferable to always omit the patch version in the documentation.

Reply:
It does, if you specify it. So if you specify `msrv = 1.30.1` and a lint is marked as `1.30.0` this lint will not trigger (We should add a test for this, though).

Reply:
> It does, if you specify it. So if you specify `msrv = 1.30.0` and a lint is marked as `1.30.1` this lint will not trigger.

Oh, I didn't realize it was possible with the current implementation. Thanks for pointing it out.

> The thing is, that features don't usually get stabilized in a patch version, so no lint will probably ever have a patch version `>0`

Yeah, this is what I actually wanted to say. ([There was a case where stabilization was reverted in a patch version due to a security vulnerability](https://github.com/rust-lang/rust/pull/60785), but I don't think there were any cases where new APIs were stabilized.)